### ISSUE TITLE
VueRecord throw a TypeError: failed to execute connect on audioNode

### ISSUE THREAD

**Issue by wyh** [codekraft-studio/vue-record#8]
The detail error stack is as following: 

```
VueRecord.umd.js?c9d0:4086 Uncaught TypeError: Failed to execute 'connect' on 'AudioNode': No function was found that matched the signature provided.
    at startContext (VueRecord.umd.js?c9d0:4086)
    at TapListener._ended (VueRecord.umd.js?c9d0:4056)
startContext @ VueRecord.umd.js?c9d0:4086
TapListener._ended @ VueRecord.umd.js?c9d0:4056

```

Following steps have been made: 

1. install: npm install @codekraft-studio/vue-record
2.  import it in the component: 

```
import Vue from 'vue'
import VueRecord from '@codekraft-studio/vue-record'

Vue.use(VueRecord)
```

3. And finally using it in the compoent:

```
<VueRecordAudio mode="hold" @result="recordAudio" />
```

**Comment by wyh** [comment]
However, the demo can be opened and it works well.

**Comment by b4dnewz** [comment]
are you using safari browser?

**Comment by wyh** [comment]
No. My  Browser is Chrome, version: 80.0.3987.122.

**Comment by jonalxh** [comment]
Same issue here.
Google Chrome Version 80.0.3987.122 (Official Build) (64-bit)

**Comment by b4dnewz** [comment]
hi, actually i'm not able to replicate the issue, but I'm quite sure it's related to an [archived package](https://github.com/tambien/StartAudioContext) we use since the last version.

I need to investigate about that and find a possible substitute for that package.

**Comment by jonalxh** [comment]
Any progress?

**Comment by b4dnewz** [comment]
not really, unfortunately I don't have enough time right now to take a look to this.
i'm still evaluating which is the best substitution for tone.js, which is causing the issue, before start a project refactoring

right now I found [MediaStreamRecorder](https://github.com/streamproc/MediaStreamRecorder) and [audio-recorder-polyfill](https://github.com/ai/audio-recorder-polyfill) which can be used to fix this

**Comment by jon85p** [comment]
Hello! 
On my company we had this error, the solution (in our case) was to deploy to https, from Chrome 74 the audio context don't work over insecure deployments (without ssl)

https://stackoverflow.com/a/58716697

Works for you? @wyh
